monitoring/harmony-monitor/node-mock-rpc/README.md

```diff
@@ -0,0 +1,22 @@
+# Mock RPC server
+This is a small polka server running in node, for the purpose of unit testing Pagerduty
```

Comment:
no, for the purpose of unit testing watchdog's check of consensus liviness. everyone knows pagerduty work, watchdog is what we are testing